.github/ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,72 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as **Ready** in the CMS Design System.
+
+## Documentation is complete
+
+**What to check:**
+- [ ] Description of component or pattern
+- [ ] HTML code snippet example(s)
+- [ ] React code snippet example(s)
+- [ ] All react properties are explained
+- [ ] General guidance or best practices
+- [ ] Usability - when to use
+- [ ] Usability - when to consider something else
+- [ ] Accessibility considerations
+
+## Validate HTML
+If you are using Google Chrome or Firefox consider getting the [axe plugin](https://www.deque.com/axe) for validation.
```

Comment:
@melwoodard encouraged me to be more opinionated on this point. So I'm going to ask that we make this a bullet point and move it to item #2 below testing on W3.org. Bullet point could say
* Install the [axe plugin](https://deque.com/axe) for Chrome or Firefox